fdbclient/CommitProxyInterface.h

```diff
@@ -327,14 +327,15 @@ struct GetRawCommittedVersionReply {
 	bool locked;
 	Optional<Value> metadataVersion;
 	Version minKnownCommittedVersion;
+	std::vector<Tag> tags;
```

Comment:
shouldn't this be of type `std::vector<std::pair<Tag, Version>>`?

Reply:
Yes, I think that will capture the overhead more accurately. Will change.

Reply:
unordered_map ?

Reply:
Done. It is an std::map in the latest commit (didn't see/remember the above std::vector/std::unordered_map recommendations before I made that change), could change it to std::vector/std::unordered_map - the specific data structure probably doesn't matter much In this context.